src/applications/mhv-medical-records/containers/LabsAndTests.jsx

```diff
@@ -82,7 +82,7 @@ const LabsAndTests = () => {
       >
         <NewRecordsIndicator
           refreshState={refresh}
-          extractType={refreshExtractTypes.CHEM_HEM}
+          extractType={refreshExtractTypes.ALLERGY}
```

Comment:
The `NewRecordsIndicator` component for Labs & Tests should have type `refreshExtractTypes.CHEM_HEM`.